examples/destination/java/src/main/java/destination/DestinationServiceImpl.java

```diff
@@ -48,26 +49,32 @@ public void configurationForm(ConfigurationFormRequest request, StreamObserver<C
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.INT).setMapTo(DestinationType.newBuilder().setName("INTEGER").setMapTo(DataType.INT).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.LONG).setMapTo(DestinationType.newBuilder().setName("LONG_INTEGER").setMapTo(DataType.LONG).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.DECIMAL).setMapTo(DestinationType.newBuilder().setName("DECIMAL").setMapTo(DataType.DECIMAL).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.FLOAT).setMapTo(DestinationType.newBuilder().setName("FLOAT").setMapTo(DataType.FLOAT).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.DOUBLE).setMapTo(DestinationType.newBuilder().setName("DOUBLE").setMapTo(DataType.DOUBLE).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.NAIVE_TIME).setMapTo(DestinationType.newBuilder().setName("TIME").setMapTo(DataType.NAIVE_TIME).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.NAIVE_DATE).setMapTo(DestinationType.newBuilder().setName("DATE").setMapTo(DataType.NAIVE_DATE).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.NAIVE_DATETIME).setMapTo(DestinationType.newBuilder().setName("DATETIME").setMapTo(DataType.NAIVE_DATETIME).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.UTC_DATETIME).setMapTo(DestinationType.newBuilder().setName("UTC_DATETIME").setMapTo(DataType.UTC_DATETIME).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.BINARY).setMapTo(DestinationType.newBuilder().setName("BLOB").setMapTo(DataType.BINARY).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.XML).setMapTo(DestinationType.newBuilder().setName("XML").setMapTo(DataType.XML).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.STRING).setMapTo(DestinationType.newBuilder().setName("VARCHAR").setMapTo(DataType.STRING).build()).build(),
-                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.JSON).setMapTo(DestinationType.newBuilder().setName("OBJECT").setMapTo(DataType.JSON).build()).build()))
+                                DataTypeMappingEntry.newBuilder().setFivetranType(DataType.SHORT).setMapTo(DestinationType.newBuilder().setName("SHORT_INTEGER").setMapTo(DataType.SHORT).setMaxValue(MaxValue.newBuilder().setNumericParam(Integer.MAX_VALUE)).build()).build(),
```

Comment:
All these examples are showing one to one mapping. Can we make a couple of them show other types of mappings, like we could map XML to "not used" and Double to Float or something..

Reply:
- made `XML` unsupported
- mapped `SHORT` and `INT` to destination `INTEGER`
- mapped `DECIMAL` AND `FLOAT` to destination `FLOAT`
- mapped `NAIVE_TIME`, `NAIVE_DATETIME`, and `UTC_DATETIME` to destination `DATETIME`